apps/common/schemas/yDaemonVaultsSchemas.ts

```diff
@@ -0,0 +1,165 @@
+		tvl_delegated: z.number(),
+		tvl: z.number(),
+		price: z.number()
+	}),
+	apy: z.object({
+		type: z.string(),
+		gross_apr: z.number(),
+		net_apy: z.number(),
+		staking_rewards_apr: z.number(),
+		fees: z.object({
+			performance: z.number(),
+			withdrawal: z.number(),
+			management: z.number(),
+			keep_crv: z.number(),
+			cvx_keep_crv: z.number()
+		}),
+		points: z.object({
+			week_ago: z.number(),
+			month_ago: z.number(),
+			inception: z.number()
+		}),
+		composite: z.object({
+			boost: z.number(),
+			pool_apy: z.number(),
+			boosted_apr: z.number(),
+			base_apr: z.number(),
+			cvx_apr: z.number(),
+			rewards_apr: z.number()
+		})
+	}),
+	details: z.object({
+		management: addressSchema,
+		governance: addressSchema,
+		guardian: addressSchema,
+		rewards: addressSchema,
+		depositLimit: z.string(),
+		availableDepositLimit: z.string(),
+		comment: z.string(),
+		apyTypeOverride: z.string(),
+		apyOverride: z.number(),
+		order: z.number(),
+		performanceFee: z.number(),
+		managementFee: z.number(),
+		depositsDisabled: z.boolean(),
+		withdrawalsDisabled: z.boolean(),
+		allowZapIn: z.boolean(),
+		allowZapOut: z.boolean(),
+		retired: z.boolean(),
+		hideAlways: z.boolean()
+	}),
+	strategies: z.array(
+		z.object({
+			address: addressSchema,
+			name: z.string(),
+			displayName: z.string(),
+			description: z.string(),
+			details: z.object({
+				keeper: addressSchema,
+				strategist: addressSchema,
+				rewards: addressSchema,
+				healthCheck: addressSchema,
+				totalDebt: z.string(),
+				totalLoss: z.string(),
+				totalGain: z.string(),
+				minDebtPerHarvest: z.string(),
+				maxDebtPerHarvest: z.string(),
+				estimatedTotalAssets: z.string(),
+				creditAvailable: z.string(),
+				debtOutstanding: z.string(),
+				expectedReturn: z.string(),
+				delegatedAssets: z.string(),
+				delegatedValue: z.string(),
+				version: z.string(),
+				protocols: z.array(z.string()).or(z.null()),
+				apr: z.number(),
+				performanceFee: z.number(),
+				lastReport: z.number(),
+				activation: z.number(),
+				keepCRV: z.number(),
+				debtLimit: z.number(),
+				withdrawalQueuePosition: z.number(),
+				doHealthCheck: z.boolean(),
+				inQueue: z.boolean(),
+				emergencyExit: z.boolean(),
+				isActive: z.boolean(),
+				debtRatio: z.number().optional()
+			}).optional(), // Optional for migratable
+			risk: z.object({
+				riskScore: z.number(),
+				riskGroup: z.string(),
+				riskDetails: z.object({
+					TVLImpact: z.number(),
+					auditScore: z.number(),
+					codeReviewScore: z.number(),
+					complexityScore: z.number(),
+					longevityImpact: z.number(),
+					protocolSafetyScore: z.number(),
+					teamKnowledgeScore: z.number(),
+					testingScore: z.number()
+				}),
+				allocation: z.object({
+					status: z.string(),
+					currentTVL: z.string(),
+					availableTVL: z.string(),
+					currentAmount: z.string(),
+					availableAmount: z.string()
+				})
+			}).optional() // Optional for migratable
```

Comment:
That's to say that the migratable vaults actually have this prop as optional, unlike the other vaults.

My comment is just to point that out, it would be required but for the migratable vaults it's actually optional.

Reply:
Yes, that's what we don't get for migratable vaults. What am I missing?